index.js

```diff
@@ -4,6 +4,9 @@ var hasFlag = require('has-flag');
 var serializeError = require('serialize-error');
 var Runner = require('./lib/runner');
 var runner = new Runner();
+var cache = require('cacha')('.ava/cache');
+
+cache.clean();
```

Comment:
> Why can't it just do that on every call for expired ttls?

Calling `clean` on every `get` call will be expensive (for example, if cache is big enough, like 1000 files) - we need to read cache directory and check every file `atime`.